docs/index.mdx

```diff
@@ -11,6 +11,11 @@ Welcome to the hackmud wiki!
 
 Use the nav bar at the top of the page to browse to the content you are looking for.
 
+#### Recommended new player guides
+
+- [[marks:Intro to the Marks System]]
```

Comment:
this link is not working correctly

Reply:
(tested via codespace)

Reply:
No links are - #489

Reply:
In my codespace, both seem to be loading fine:
![image](https://github.com/user-attachments/assets/8c26643c-2386-4169-9f45-eabf24bb3d25)